convene-web/app/views/workspaces/_room_card.html.erb

```diff
@@ -1,4 +1,4 @@
-<li class="col-span-1 bg-white rounded-lg shadow mb-4" id="<%= room.name %>" >
+<li class="col-span-1 bg-white rounded-lg shadow mb-4 <%= "--#{room.access_level}" %>" id="<%= room.name %>">
```

Comment:
Not-blocking: Something that may be nice would be to also include a `room-card` class to the `li` so we can key off of that as well.

Reply:
👍 Will be working on this on a separate PR.